packages/handlers/openapi/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@graphql-mesh/openapi",
-  "version": "0.31.2",
```

Comment:
@gilgardosh Let's keep the version here, and rename `new-openapi` to `openapi` in the changesets. It will be released exactly how we wanted.